python/arcticdb/version_store/_normalization.py

```diff
@@ -1070,21 +1070,8 @@ def _msgpack_unpackb(self, buff, raw=False):
 
 class Pickler(object):
     @staticmethod
-    def read(data, pickled_in_python2=False):
-        if isinstance(data, str):
-            return pickle.loads(data.encode("ascii"), encoding="bytes")
-        elif isinstance(data, str):
```

Comment:
This else case was clearly wrong https://github.com/man-group/ArcticDB/pull/2156#issuecomment-2624547655